<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgSmartReports</class>
 <widget class="QDialog" name="DlgSmartReports">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reports</string>
  </property>
  <property name="bigfont" stdset="0">
   <string>1</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frameReports">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Date start</string>
        </property>
        <property name="middlefont" stdset="0">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="C5DateEdit" name="leDateStart">
        <property name="middlefont" stdset="0">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Date end</string>
        </property>
        <property name="middlefont" stdset="0">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="C5DateEdit" name="leDateEnd">
        <property name="middlefont" stdset="0">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Time start</string>
        </property>
        <property name="middlefont" stdset="0">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="C5LineEdit" name="leTimeStart">
        <property name="inputMask">
         <string>##:##</string>
        </property>
        <property name="text">
         <string>00:00</string>
        </property>
        <property name="middlefont" stdset="0">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Time end</string>
        </property>
        <property name="middlefont" stdset="0">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="C5LineEdit" name="leTimeEnd">
        <property name="inputMask">
         <string>##:##</string>
        </property>
        <property name="text">
         <string>23:59</string>
        </property>
        <property name="middlefont" stdset="0">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="4">
       <widget class="QListWidget" name="list">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="bigfont" stdset="0">
         <string>1</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>C5LineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">c5lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>C5DateEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">c5dateedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
